shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -55,9 +58,29 @@ public void createBridgeExecutor(ProcessorDTO processorDTO) {
                 .create(BridgeExecutor.fromDTO(processorDTO, namespace.getMetadata().getName(), executorImage));
     }
 
+    @Override
+    public ConfigMap fetchOrCreateBridgeExecutorProcessorConfigMap(BridgeExecutor bridgeExecutor) {
+        ConfigMap configMap = kubernetesClient.configMaps().inNamespace(bridgeExecutor.getMetadata().getNamespace()).withName(bridgeExecutor.getMetadata().getName()).get();
+
+        if (configMap != null) {
+            LOGGER.info("Executor processor ConfigMap for BridgeProcessor: '{}' in namespace '{}' is already present.", bridgeExecutor.getMetadata().getName(),
+                    bridgeExecutor.getMetadata().getNamespace());
+            return configMap;
+        }
+
+        configMap = templateProvider.loadBridgeExecutorProcessorConfigMapTemplate(bridgeExecutor);
+        try {
+            configMap.getData().replace("processor.json", new ObjectMapper().writeValueAsString(bridgeExecutor.toDTO()));
```

Comment:
At the minute, we store the `ProcessorDTO` inside the configmap, then the executor deserializes it so to have all the necessary information about the processor definition and the bridgeId events to consume